NOTICE.txt

```diff
@@ -0,0 +1,31 @@
+Elasticsearch .NET Transport
+Copyright 2012-2018 Elasticsearch B.V.
```

Comment:
We'll want to update these dates to include any years where significant (copyrightable) changes were made to this project.

Reply:
I've updated the end date on this.